tests/testMultiTypeTable.m

```diff
@@ -48,13 +48,13 @@
     end
 
     properties
-        exampleTable;           % Example Multi-Type Table for testing
-        initialTypesTable;
-        initialAllowedTypes = {'constant'  'data'  'function', 'gaussian'};
-        initialAllowedActions = {'add'  'subtract'};
-        initialTypesCount = 1;
-        initialTypesAutoNameCounter = 1;
-        initialTypesAutoNameString = 'Row';
+        exampleTable            % Example Multi-Type Table for testing
```

Comment:
I don't think there is any benefit to semicolon so we  have to agree a style an unify all

Reply:
I'd spotted this as well. I'll take the semicolons out, which I prefer.